README.md

```diff
@@ -1,3 +1,45 @@
+### Wallapop FIX
+
+In order to avoid that our legacy code use Genson serializer we delete the "magic":
```

Comment:
In order to avoid that magically some system start to use Genson serializer in undesired places  we have removed the "magic":